<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Porkplayer Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="vb_1" stretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QGridLayout" name="gd_1" columnstretch="1,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="1" column="1">
        <widget class="QComboBox" name="s_colorSpaceBox">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="s_dataLevelBox">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="s_eotfBox">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="s_dataLevelText">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Data Level :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="s_displaySettingText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Target Display</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="s_eotfText">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>EOTF :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="s_colorSpcaeText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Color Space :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="s_ffmpegPathText">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gd_2" columnstretch="1,0">
       <item row="0" column="0">
        <widget class="QLabel" name="s_exportSettingText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="s_ffmpegPathTitle">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FFmpeg Path :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="s_ffmpegPathEdit">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="s_applyBtn">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
